css/main.scss

```diff
@@ -486,6 +486,38 @@ i:hover {
   }
 }
 
+.cookie-banner {
+  width: 100%;
+  padding: 20px 0;
+  position: fixed;
+  bottom: 0;
+  z-index: 9999;
+  background-color: #6c2580;
+
+  p {
+    margin: 0;
+    color: #fff;
+  }
+
+  a {
+    color: #fff;
+  }
+}
+
+.cookie-button {
+  display: inline-block;
+  border-width: 2px;
+  border-style: solid;
+  padding: 5px 50px;
+  cursor: pointer;
+
+  &:hover {
+    background-color: #e54e22;
```

Comment:
This looks strange: 
<img width="233" alt="bildschirmfoto 2018-01-24 um 15 12 36" src="https://user-images.githubusercontent.com/1510/35336542-0d8b7c50-0119-11e8-8a85-261d2a4c8834.png">